ShopAppTests/App/Cart/Views/Cells/TableView/CartTableViewCellSpec.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  CartTableViewCellSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 3/9/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class CartTableViewCellSpec: QuickSpec {
```

Comment:
Add tests for populatePricePerOne in line 102

Reply:
This method accepts non optional property, but return optional. I haven't any ideas how to test this line